doc/Migration.md

```diff
@@ -19,6 +19,26 @@ For example:
 }
 ```
 
+### v1.28.0
```

Comment:
Please update this to `1.29.0`.

Reply:
Thank you for the review, @msujew! I updated the version.